cfgmgr/natmgr.cpp

```diff
@@ -5049,7 +5049,7 @@ void NatMgr::doStaticNatTask(Consumer &consumer)
         /* Ensure the key size is 1 otherwise ignore */
         if (keys.size() != STATIC_NAT_KEY_SIZE)
         {
-            SWSS_LOG_ERROR("Invalid key size %lu, skipping %s", keys.size(), key.c_str());
+            SWSS_LOG_ERROR("Invalid key size %u, skipping %s", keys.size(), key.c_str());
```

Comment:
for size_t or size_type "%z" should be used. Can you please verify it

Reply:
%z is not supported. I changed it to PRIu32.

Reply:
Changed it to %zu.